docs/source/examplecleanup.rst

```diff
@@ -0,0 +1,116 @@
+Would you like to know more?
+============================
+
+End-to-End Example
+------------------
+
```

Comment:
Can you add something up top here, along the lines of:

> In order to follow along with this guide, you will also want to install the `BFG` tool from ...

Just talking about BFG without any kind of introduction is a bit off-putting.